simpinkscr/simple_inkscape_scripting.py

```diff
@@ -3724,7 +3724,8 @@ def effect(self):
 
         # Remove an unnecessary import that may be introduced when
         # running from Visual Studio Code.
-        code.replace("from simpinkscr import *", "")
+        pattern = r"^[ \t]*(import\s+(inkex|simpinkscr).*|(from\s+(inkex|simpinkscr)\s+import).*)[\r\n]?"
```

Comment:
Why is the `?` needed at the end?  Don't you always want to consume the end-of-line character?

Reply:
Well, I was testing on a regex tool and the last line didn't have an EOL, so I didn't really think about it.  I assume there is no value if someone did end a file with an import, so I will remove it for clarity.